docs_website/docs/integrations/add_dag_exporter.md

```diff
@@ -0,0 +1,33 @@
+---
+id: add_dag_exporter
+title: DAG Exporter
+sidebar_label: DAG Exporter
+---
+
+## What is a dag exporter?
+
+DAG exporter is a way to create a workflow from Query Cells in a Datadoc.
+
+## Implementation
+
+To keep the process standardized, please create a dag exporter under <project_root>/querybook/server/lib/export/dag_exporters. All dag exporters must inherit from BaseDAGExporter that lives in <project_root>/querybook/server/lib/export/base_dag_exporter.py.
```

Comment:
mention how to do it via plugins, i imagine it is likely rare to add dag exporter to open source repo directly